### DISCUSSION TITLE
能否加入默认宽屏播放 选项卡？

### DISCUSSION THREAD

**Discussion by onehardlearner** [the1812/Bilibili-Evolved#1612] [功能建议 / Ideas]
如题，先谢过大佬了，插件很好用，敏感肌也能用.jpg

**Comment by LonelySteve** [comment]
第一个设置就是啊（

![image](https://user-images.githubusercontent.com/34429322/109787276-05ff3400-7c49-11eb-8c2d-7e902c9d9eed.png)
